src/tools/mesh_tester.cpp

```diff
@@ -127,53 +130,65 @@ const std::set<RuntimePolicy> Input::s_validPolicies({
 
 void Input::parse(int argc, char** argv, CLI::App& app)
 {
-  app.add_option("-r,--resolution", resolution,
-                 "Resolution of uniform grid. \n"
-                 "Set to 1 to run the naive algorithm (without a spatial index). \n"
-                 "Set to less than 1 to use the spatial index with a resolution \n"
-                 "of the cube root of the number of triangles.")
-  ->capture_default_str();
-
-  app.add_option("-p, --policy", policy,
-                 "With \'-r 1\', set runtime policy. \n"
-                 "Set to 0 to use the sequential algorithm (w/o RAJA). \n"
-  #ifdef AXOM_USE_RAJA
-                 "Set to 1 to use the RAJA sequential policy. \n"
-    #ifdef AXOM_USE_OPENMP
-                 "Set to 2 to use the RAJA OpenMP policy. \n"
-    #endif
-    #ifdef AXOM_USE_CUDA
-                 "Set to 3 to use the RAJA CUDA policy."
-    #endif
+  app
```

Comment:
This one is a bit strange.

Reply:
Thanks.  I'll manually edit this one, as discussed in the overview meeting.

Reply:
@kennyweiss did you try pulling the macro logic out of the function call and saving what it generates to a string that can be passed to the function?

Reply:
Yes -- here is what it looks like now:
https://github.com/LLNL/axom/blob/e43484a7d0d482aa9d16ac5caac5b835f1a4c561/src/tools/mesh_tester.cpp#L147-L162